crates/vm/levm/runner/README.md

```diff
@@ -0,0 +1,24 @@
+## LEVM Runner
+
+Summary: You can send a transaction directly to the EVM specifying things like fork, bytecode, pre-state and setting an initial stack and memory for testing/benchmarking purposes. After execution it will print the final stack and memory, state diff and execution result.
+
+Example Run: `cargo run -- --input input_example.json --code code_example.txt`
+
+The program expects 2 inputs:
+- One JSON with fields like the Transaction, Fork, etc. These are all specified in `input_example.json`, you can copy that for you input and make changes.
+- Bytecode in mnemonic, separated by spaces or newlines. Example in `code_example.txt`
+
+You can provide input, bytecode or both. If input is not provided default values will be used. If bytecode file is send as argument and the transaction is a `CALL` then it will be the bytecode of the contract being called; but if transaction is `CREATE` then it will be the initcode.
```

Comment:
```suggestion
You can provide input, bytecode or both. If input is not provided default values will be used. If bytecode file is sent as argument and the transaction is a `CALL` then it will be the bytecode of the contract being called; but if the transaction is `CREATE` then it will be the initcode.
```